testsuite/run_sets/container_init_clients.yml

```diff
@@ -0,0 +1,11 @@
+
+## Container features BEGIN ###
+
+# IMMUTABLE ORDER
+- features/init_clients/allcli_update_activationkeys.feature
```

Comment:
This feature might not be needed if activation keys are not used with containers.